docs/postgres.md

```diff
@@ -6,11 +6,8 @@ describes how it can be configured.
 
 ## Building LND with postgres support
 
-To build LND with postgres support, include the following build tag:
-
-```shell
-⛰  make tags="kvdb_postgres"
-```
+To build LND with postgres support and for your specific platform, edit https://github.com/lightningnetwork/lnd/blob/1abe1da5b3a0e914b8c0f0d837fc0c17bed74a88/make/release_flags.mk#L41 by adding the `kvdb_postgres` to the end of the line.
```

Comment:
This is now done by default. Perhaps the command should just say `make release-install` which includes all required tags.